Azure/azure-cli
# [ACR] Add new command to check if specific blob is available to pull from registry

**Description**
- Add new command to check if blobs within given image can be pulled from repository using command `az acr check-health --name myRegistry --image hello-world@sha256@abc123`

**Testing Guide**
- Verify if blobs in image 'sha256:abc123' can be pulled from the repository: `az acr check-health -n MyRegistry --image hello-world@sha256:abc123`
- Verify if blobs referenced by tag 'latest' can be pulled from the repository: `az acr check-health -n MyRegistry -t hello-world:latest`

**History Notes**
[ACR] `az acr check-health --name --image`: Add new command to check if blobs in given image can be pulled from registry

---

This checklist is used to make sure that common guidelines for a pull request are followed.
